MekHQ/src/mekhq/gui/adapter/PersonnelTableMouseAdapter.java

```diff
@@ -1210,9 +1229,7 @@ protected Optional<JPopupMenu> createPopupMenu() {
         JMenuHelpers.addMenuIfNonEmpty(popup, new AssignPersonToUnitMenu(gui.getCampaign(), selected));
 
         if (oneSelected && person.getStatus().isActive()) {
-            if (gui.getCampaign().getCampaignOptions().isUseManualMarriages()
-                    && (gui.getCampaign().getMarriage().canMarry(gui.getCampaign(),
-                            gui.getCampaign().getLocalDate(), person, false) == null)) {
+            if (gui.getCampaign().getCampaignOptions().isUseManualMarriages() && (gui.getCampaign().getMarriage().canMarry(gui.getCampaign(), gui.getCampaign().getLocalDate(), person, false) == null)) {
```

Comment:
Needs to be formatted for easier reading.

Reply:
Gah, I must have mucked something up when fixing the bad merge, because I didn't manually make these changes.

I'll make sure to go through the whole file and check to ensure there aren't more mess-ups. Thanks for catching this!